editor.planx.uk/src/pages/FlowEditor/components/Settings/GeneralSettings/TrialAccountForm.tsx

```diff
@@ -62,11 +62,14 @@ export default function TrialAccountForm({
             }
           />
           <SettingsDescription>
-            <p>Toggle this team between "trial" access and full access</p>
+            <p>Toggle this team between "trial" access and full access.</p>
             <p>
               A trial team had limited access to PlanX features - they are not
               able to turn services online.
             </p>
+            <p>
+              Toggling this to a trial account will turn this team's flows offline.
+            </p>
```

Comment:
This can be tested by toggling a team to and from trial account status.

The query where `toggle_flows_offline: true` will return a number of `affected_rows` (flows toggled offline). This will not be present when a service is promoted back to full access.

**Toggling to trial account**
![image](https://github.com/user-attachments/assets/4bc425c4-ccde-4462-a04b-d9cb3810cef3)

**Toggling back to full access**
![image](https://github.com/user-attachments/assets/9bd862c7-18b1-4468-b721-8b55fb1e7fdd)

Toggling a team back to full access will not restore the previous online state of flows, this will need to be manually handled.